Vagrantfile

```diff
@@ -33,6 +34,7 @@ Vagrant.configure("2") do |config|
     apt-get install -y curl
     curl -sL https://deb.nodesource.com/setup | sudo bash -
     sudo apt-get install -y nodejs
+    npm install nodemon
```

Comment:
I think nodemon should be installed with the `-g` option. See https://www.npmjs.com/package/nodemon#installation